--- origsrc/util-macros-1.1.5/xorg-macros.m4	2006-12-23 22:17:23.000000000 -0600
+++ src/util-macros-1.1.5/xorg-macros.m4	2007-08-08 01:22:20.000000000 -0500
@@ -80,7 +80,7 @@
 		RAWCPPFLAGS=-undef
 		AC_MSG_RESULT([yes])
 	else
-		AC_MSG_ERROR([${RAWCPP} defines unix with or without -undef.  I don't know what to do.])
+		AC_MSG_RESULT([no])
 	fi
 fi
 rm -f conftest.$ac_ext
@@ -94,7 +94,7 @@
 		RAWCPPFLAGS="${RAWCPPFLAGS} -traditional"
 		AC_MSG_RESULT([yes])
 	else
-		AC_MSG_ERROR([${RAWCPP} does not preserve whitespace with or without -traditional.  I don't know what to do.])
+		AC_MSG_RESULT([no])
 	fi
 fi
 rm -f conftest.$ac_ext
--- origsrc/util-macros-1.1.5/xorg-macros.m4.in	2006-11-08 08:23:10.000000000 -0600
+++ src/util-macros-1.1.5/xorg-macros.m4.in	2007-08-08 01:21:18.078125000 -0500
@@ -80,7 +80,7 @@
 		RAWCPPFLAGS=-undef
 		AC_MSG_RESULT([yes])
 	else
-		AC_MSG_ERROR([${RAWCPP} defines unix with or without -undef.  I don't know what to do.])
+		AC_MSG_RESULT([no])
 	fi
 fi
 rm -f conftest.$ac_ext
@@ -94,7 +94,7 @@
 		RAWCPPFLAGS="${RAWCPPFLAGS} -traditional"
 		AC_MSG_RESULT([yes])
 	else
-		AC_MSG_ERROR([${RAWCPP} does not preserve whitespace with or without -traditional.  I don't know what to do.])
+		AC_MSG_RESULT([no])
 	fi
 fi
 rm -f conftest.$ac_ext
